examples/textarea/index.tsx

```diff
@@ -0,0 +1,30 @@
+import * as Ariakit from "@ariakit/react";
+import * as React from "react";
+import "./style.css";
+
+export default function Example() {
+  return (
+    <label className="label">
+      Comment
+      <Textarea
+        className="textarea"
+        placeholder="Write your comment, be kind"
+      />
+    </label>
+  );
+}
+
+interface TextareaProps extends Ariakit.FocusableProps<"textarea"> {}
+
+export const Textarea = React.forwardRef<
+  React.ElementRef<"textarea">,
+  TextareaProps
+>((props, forwardedRef) => {
+  return (
+    <Ariakit.Role.textarea
+      {...props}
+      ref={forwardedRef}
+      render={<Ariakit.Focusable render={props.render || <textarea />} />}
+    />
+  );
+});
```

Comment:
Suggestions based on [Name functions inside forwardRef](https://ariakit-82ilivk9y-ariakit.vercel.app/guide/coding-guidelines#name-functions-inside-forwardref):

```suggestion
export const Textarea = React.forwardRef<
  HTMLTextAreaElement,
  TextareaProps
>(function Textarea(props, ref) {
  return (
    <Ariakit.Role.textarea
      {...props}
      ref={ref}
      render={<Ariakit.Focusable render={props.render || <textarea />} />}
    />
  );
});
```

Also, I think it would be more readable if it used named imports:

```js
import { Role, Focusable } from "@ariakit/react";
```

See [Import namespace](https://ariakit-82ilivk9y-ariakit.vercel.app/guide/coding-guidelines#import-namespace)

Lastly, since this implementation is the focus of the example, it can be placed above the `Example` function.

Reply:
https://github.com/ariakit/ariakit/pull/4032/commits/04757bc07e110e47236d7951668d45f47e91937c: rewrote the example per suggestions